### DISCUSSION TITLE
Viernes Noche - Material Clase 11 (26/06)

### DISCUSSION THREAD

**Discussion by IanMartin94** [dds-jv/foro#151] [Consultas generales] [answered]
Buenas,

Les consulto ya que no veo cargada la bitacora de la clase del 26/06. 

Cuando quiero acceder, me tira un 404. Por favor, ¿Podrán subirla?

Desde ya muchas gracias.

Saludos.

**Comment by gastonprieto** [comment, marked as answer]
Buenas!

@roli93 despues te fijas.

@IanMartin94 te dejo el [link](https://us02web.zoom.us/rec/share/3PF3L6372TlOH6vPtmuHR65_Mo_Paaa80XdMr6Vfzxz9b2Fxn2v7iPw_o35j8MG6?startTime=1593209247000) al video de la clase:

**Comment by IanMartin94** [comment]
Genial, muchas gracias !
